move_base_msgs/package.xml

```diff
@@ -8,21 +10,23 @@
     </description>
     <author>Eitan Marder-Eppstein</author>
     <author email="contradict@gmail.com">contradict@gmail.com</author>
-    <maintainer email="davidvlu@gmail.com">David V. Lu!!</maintainer>
-    <maintainer email="mferguson@fetchrobotics.com">Michael Ferguson</maintainer>
+    <author email="davidvlu@gmail.com">David V. Lu!!</author>
+    <author email="mferguson@fetchrobotics.com">Michael Ferguson</author>
```

Comment:
@DLu @mikeferguson I moved you guys to be authors on this ROS 2 specific branch (we're re-enabling this package just now), if you'd like to stay maintainers let me know and I'll undo it. I'm just trying to save you guys some emails if this were to fail to build or something on the buildfarm.